### PR TITLE
{vis}[foss/2018b] pygraphviz v1.5 w/ Python 3.6.6 + Graphviz v2.40.1, GTS v0.7.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3142.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/6479cc1a35c79a8667d4244ee85ded23 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2405.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ab66a011ce7be3211b1741c777d69bde for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2001.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/215c3fa751f25f08a10d8edd481cb4a4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3142.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/34488d76c1223b87017cdaff5fba41fd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2405.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/52f8684432e9ea3f0b20e1477d046d99 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2001.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3324db07231f23b3e0604e480a1febf7 for a full test report.

File: easybuild/easyconfigs/g/Graphviz/Graphviz-2.40.1-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    ('Java', '1.8', '', True),
+    ('libpng', '1.6.34'),
+    ('Pango', '1.42.4'),
+    ('Perl', '5.28.0'),
+    ('Qt5', '5.10.1'),
+    ('Tcl', '8.6.8'),
+    ('zlib', '1.2.11'),
+]
+
+builddependencies = [
```
Comment (akesandgren) [inline easybuild/easyconfigs/g/Graphviz/Graphviz-2.40.1-foss-2018b.eb]: Don't we usually put builddes before deps?

**Comment by boegel** [comment]
@akesandgren good to go?

File: easybuild/easyconfigs/p/pygraphviz/pygraphviz-1.5-foss-2018b-Python-3.6.6.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+easyblock = 'PythonPackage'
+
+name = 'pygraphviz'
+version = '1.5'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://pygraphviz.github.io/'
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/pygraphviz/pygraphviz-1.5-foss-2018b-Python-3.6.6.eb]: https

Comment (boegel) [reply easybuild/easyconfigs/p/pygraphviz/pygraphviz-1.5-foss-2018b-Python-3.6.6.eb]: fixed in a60acb7790

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 2 out of 4 (3 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/e06888b8323183682c9497f38862110a for a full test report.

**Comment by boegel** [comment]
@akesandgren Any idea what causes that issue?

**Comment by boegel** [comment]
(re-triggering Travis checks...)

**Comment by boegel** [comment]
superseded by #9969 and (partially) by #9782, so closing...

**Pull request closed without merging**